pyanaconda/modules/storage/devicetree/root.py

```diff
@@ -353,3 +354,22 @@ def name(self):
     def device(self):
         """The root device or None."""
         return self.mounts.get("/")
+
+    def copy(self, storage):
+        """Create a copy with devices of the given storage model.
+
+        :param InstallerStorage storage: a storage model
+        :return Root: a copy of this root object
+        """
+        new_root = copy.deepcopy(self)
+
+        def _get_device(d):
+            return storage.devicetree.get_device_by_id(d.id, hidden=True)
+
+        def _get_mount(i):
+            m, d = i[0], _get_device(i[1])
```

Comment:
Could you please do the assignment on two lines. It's better to read (at least for me) :D.